requirements.in

```diff
@@ -18,6 +18,8 @@ mypy
 pytest
 pytest-cov
 pyupgrade
-safety
+ruamel-yaml; python_version < '3.11'  # dependency of safety
+ruamel-yaml-clib; python_version < '3.11'  # indirect dependency of safety
+safety; python_version < '3.11'
```

Comment:
This was necessary because [ruamel-yaml-clib doesn't have wheels](https://pypi.org/project/ruamel.yaml.clib/#files) for Python 3.11 on Linux.  Installing gcc etc. just to be able to install safety and its dependencies seems pointless when it's not even used.  (The GitHub Actions step has a `if: matrix.python-version == '3.9'`.)

Maybe move linters and formatters to a `requirements-lint.in` and use `requirements.txt` as a constraint?  This would make it possible to skip unnecessary installs outside of Python 3.9 altogether.